### PR TITLE
[JBWS-4308] chapter edits

### PR DESCRIPTION
**Pull request by rsearls**

### CODE REVIEW

File: docbook/src/main/doc/adoc/content/chapter-1-Web_Services_Introduction.adoc [hunk 1]
```diff
@@ -143,18 +117,12 @@ http://www.w3.org/TR/soap/[SOAP] specification. This means XML messages
...
+The above specifications are quite common nowadays in the IT
+industry. Many enterprises have used them for years.
 
-However a real added value to the web service platform comes from a lot
-of recent additional specifications. These cover features that are
-really relevant to deliver mission critical enterprise services. For
-example some of the most important agreements major vendors came to are
-those on security (WS-Security) and reliable messaging (WS-Reliable
-Messaging).
+In resent years specifications have been created for security (WS-Security) and reliable messaging (WS-Reliable Messaging).
```
Comment (jimma) [inline docbook/src/main/doc/adoc/content/chapter-1-Web_Services_Introduction.adoc:127]: typo? should be recent ?

File: docbook/src/main/doc/adoc/content/chapter-2-Quick_Start.adoc [hunk 2]
```diff
@@ -1,26 +1,21 @@
 == Quick Start
 
 JBossWS uses WildFly as its target container. The following examples
-focus on web service deployments that leverage EJB3 service
+focus on web service features that leverage EJB3 service
 implementations and the JAX-WS programming models. For further
-information on POJO service implementations and advanced topics you need
-consult the <<Jakarta XML Web Services User Guide>>.
+information on POJO service implementations and advanced topics
+consult the https://docs.jboss.org/author/display/JBWS/JAX-WS%20User%20Guide.html[Web Services User Guide].
```
Comment (jimma) [inline docbook/src/main/doc/adoc/content/chapter-2-Quick_Start.adoc:7]: ``<<Jakarta XML Web Services User Guide>>`` can be automatically transformed to the correct chapter link. Please use this format for the internal chapter link.

**Review by jimma — COMMENTED** [review]
Thanks for the huge improvement for the doc @rsearls . Except for the minor issue, it looks good.

File: docbook/src/main/doc/adoc/content/chapter-1-Web_Services_Introduction.adoc (hunk 1 above)
Comment (rsearls) [reply docbook/src/main/doc/adoc/content/chapter-1-Web_Services_Introduction.adoc]: done

File: docbook/src/main/doc/adoc/content/chapter-2-Quick_Start.adoc (hunk 2 above)
Comment (rsearls) [reply docbook/src/main/doc/adoc/content/chapter-2-Quick_Start.adoc]: done

**Pull request merged**